services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinematically_id: int
+) -> QuerySet:
```

Comment:
Remember:`.all()`, `.filter()` returns QuerySet`
.get()`, `.delete()`, `.create()` returns model class, `-> User` for. example

Fix it everywhere